src/package/natvis/cppwinrtvisualizer.vcxproj

```diff
@@ -76,23 +77,23 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <LinkIncremental>true</LinkIncremental>
-    <OutDir>x86\$(Configuration)\</OutDir>
-    <IntDir>x86\$(Configuration)\</IntDir>
+    <OutDir>x86\$(CmakeConfiguration)\</OutDir>
```

Comment:
how does this help?  aren't the visualizer and vsix projects still building into RelWithDebInfo, which breaks the azdo pipeline?

Reply:
I thought we'd just use the names `Debug` and `Release` throughout.

Reply:
In this case, `CmakeConfiguration` can be configured by CMake, which would be set to `RelWithDebInfo`, which would match what ADO is expecting. All moot now, but that was the idea

Reply:
oh moot now, got it